xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -121,8 +122,8 @@ This constructor is protected. It can only be called from the <xref:System.Runti
         <Parameter Name="isCollectible" Type="System.Boolean" Index="0" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="isCollectible">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="isCollectible">Enable garbage collection.</param>
```

Comment:
```suggestion
        <param name="isCollectible"><see langword="true" /> to enable garbage collection; otherwise, <see langword="false" />.</param>
```